### PR TITLE
Cbcccm 153

### PR DESCRIPTION
**Pull request by jjstratton**
# Description

This PR includes the following proposed change(s):

CBCCCM 153 - updated tagging with environment names and added test and prod

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/promote-image-to-env.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      - name: Login to Openshift Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.OPENSHIFT4_REGISTRY }}
+          username: ${{ secrets.OPENSHIFT4_SA_USERNAME }}
+          password: ${{ secrets.OPENSHIFT4_SA_PASSWORD }}
+
+      - name: Pull Dev Image 
+        run: |  
+          docker pull ${{ secrets.OPENSHIFT4_REGISTRY }}/${{ secrets.LICENSE_PLATE }}-tools/cccm-api:dev
```
Comment (ChrisHoban) [inline .github/workflows/promote-image-to-env.yml:42]: At some point we may want to separate our promotions by service, but today is not that day :D Promoting everything at once is totally fine for now :)

**Review by ChrisHoban — APPROVED** [review]
Looks good :D

File: .github/workflows/promote-image-to-env.yml (hunk 1 above)
Comment (jjstratton) [reply .github/workflows/promote-image-to-env.yml:42]: yeah I have been thinking about that too! Kind of not necessary to pull and push each one every time. Ill make a ticket for it :)

**Comment by ChrisHoban** [comment]
It looks good, simple way to promote everything to Test or Prod :)

**Pull request merged**